rllib/connectors/action/pipeline.py

```diff
@@ -28,7 +28,6 @@ def __call__(self, ac_data: ActionConnectorDataType) -> ActionConnectorDataType:
             timer = self.timers[str(c)]
             with timer:
                 ac_data = c(ac_data)
-            timer.push_units_processed(1)
```

Comment:
wait, we shouldn't get rid of these? same below.

Reply:
I realized we actually don't need these after kourosh asked me about them -> https://github.com/ray-project/ray/pull/31693#discussion_r1072482664

I just took the following screenshot just to make sure that the timer actually works as expected.
When calling .mean(), the timer does not care of processed units - we don't need it.
<img width="699" alt="Screenshot 2023-01-18 at 15 54 33" src="https://user-images.githubusercontent.com/9356806/213323120-ddc90bb1-5baa-4a64-bf98-285072353ad4.png">

Reply:
oh that's right, it's part of the with statements.